datasync/chaindatafetcher/chaindata_fetcher.go

```diff
@@ -262,13 +262,38 @@ func (f *ChainDataFetcher) stopRangeFetching() error {
 	return nil
 }
 
+func (f *ChainDataFetcher) retryMakeChainEvent(blockNumber uint64) (blockchain.ChainEvent, error) {
```

Comment:
```suggestion
func (f *ChainDataFetcher) makeChainEventWithRetry(blockNumber uint64) (blockchain.ChainEvent, error) {
```
`retryMakeChainEvent` sounds like a chain event already occured.

Reply:
I agreed. Reflected :)